core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -66,6 +67,7 @@ private[spark] class EventLoggingListener(
 
   private val shouldCompress = sparkConf.getBoolean("spark.eventLog.compress", false)
   private val shouldOverwrite = sparkConf.getBoolean("spark.eventLog.overwrite", false)
+  private val shouldLogBlockUpdates = sparkConf.getBoolean("spark.eventLog.blockUpdates", false)
```

Comment:
Please use a config constant (see `internal/config/package.scala`).

Reply:
@vanzin The reason I didn't is because all of the other config settings nearby don't. Would it make sense to change the other event log settings as part of this?

Reply:
The other configs predate the config library; it's ok either way for those, but new configs should use the new library.

Reply:
Thanks for clarifying. I opted to convert other event log settings as well for uniformity.